L44-python-rich-status.md

```diff
@@ -0,0 +1,474 @@
+New RPC Status API for Python
+----
+* Author(s): lidizheng
+* Approver: ericgribkoff
+* Status: Draft
+* Implemented in: Python
+* Last updated: 12-10-2018
+* Discussion at: https://groups.google.com/forum/#!topic/grpc-io/8Ys6ba8gpLc
+
+## Abstract
+
+The current design of status API doesn't support rich status. Also the concept of status "details" doesn't comply with the gRPC spec. So, after discussion with @gnossen and @ericgribkoff, we think we should add a new set of status APIs to correct them.
```

Comment:
AFAIK rich status is not a term used elsewhere, and would need to be defined. I would prefer something more general here, such as:

> The current design of the gRPC Python servicer API for RPCs terminated with a non-OK status does not easily allow pairing the failure with associated additional information not expressible by the `grpc-status` and `grpc-message` fields. This proposal adds a rich `Status` class to gRPC Python that allows coupling the failure's status code, error string, and arbitrary additional information about the failure (sent as trailing metadata) in a unified interface.

(and please remove the second sentence about "after discussions with")

Reply:
Updated to your version. Thank you for making it more clear.